### ISSUE TITLE
Ensure accessibility

### ISSUE THREAD

**Issue by james-minton** [Hyperobjekt/seeing-solitary#50]
This site must follow WCAG 2.1 Accessibility guidelines; developer practices for ensuring conformance to these are [outlined here in our docs.](https://github.com/Hyperobjekt/docs/blob/main/developer-guidelines.md#accessibility-guidelines)

All developers should follow these norms & practices while coding to prevent the need for time-consuming fixes later; furthermore, we should explore creating an automated accessibility review system for each PR.

To do:
 - [ ] Explore creating an Action to review PRs for accessibility. [Here is an example of one such setup we might emulate](https://www.adrianbolonio.com/blog/accessibility-github-actions).
 - [ ] Review PRs for conformance and if they fail, either
 - direct developers to correct errors
 - note in the PR why an exception can be made
 - if the error creates a significant bottleneck and we need to merge anyway, note the issue in a separate ticket to be resolved before launch

**Comment by james-minton** [comment]
@cfhull does an automated accessibility check as described above make sense, and if so, would you be able to set up the action to run it?

**Comment by cfhull** [comment]
Yes. I'll also make sure our eslint config has accessibility rules enabled if it doesn't already.

**Comment by lizlove** [comment]
Some packages I've before to check for accessibility in React: 
- https://www.npmjs.com/package/react-a11y
- https://www.npmjs.com/package/eslint-plugin-jsx-a11y
More high-powered testing engine for accessibility:
- https://www.npmjs.com/package/axe-core

**Comment by cfhull** [comment]
I had better luck using [pally-ci](https://github.com/pa11y/pa11y-ci) since it had better support for running as a GitHub action, and also ran faster. There's only one error it's currently throwing, and it's for the page below.

The error is that the contrast ratio between the teal hex (#69a2a2) and the text (#ffffff) is too low. The tool recommended the color I set for Maine (#78746e)

`This element has insufficient contrast at this conformance level. Expected a
   contrast ratio of at least 4.5:1, but text in this element has a contrast
   ratio of 1.08:1. Recommendation:  change background to #78746e.`


<img width="275" alt="image" src="https://user-images.githubusercontent.com/5115526/194556630-cecb33ff-b59d-4cbb-9f59-97767cdc8802.png">

**Comment by james-minton** [comment]
@cfhull @lizlove good catch -- actually though, that hexbin map looks like it's from an earlier commit; the more recent state of `development` look like the image below (closer to the mockup).

I think we're going to get in trouble here on contrast ratio, though, with the white text against yellow/orange backgrounds. But in the mockups, we are using a larger font size and a drop shadow on the text, which helps with contrast.

I think we're going to have to do a combination of implementing those elements from the mockup, plus making a special statement in our accessibility report about why this deserves an exception. Between the mockup treatment and the fact that reading the hexbin isn't the only way to get the data you need (there will be a tooltip, plus dedicated pages for each jurisdiction), we'll be fine.

But just a heads-up, this hexbin may continue to fail automated checks because it's just a blunt-instrument algorithm to check the contrast ratio. It's okay to ignore those as long as we add the drop-shadow, etc.

<img width="1209" alt="Screen Shot 2022-10-07 at 9 00 48 AM" src="https://user-images.githubusercontent.com/33635008/194561619-13bf3c58-a7be-411f-acab-d4dc50501828.png">

**Comment by cfhull** [comment]
I didn't release my remote was pointing at a fork, so I wasn't pulling the latest development. I ran it again and there's clearly an issue with accessibility tools reading these SVGs since it still is saying a ratio of 1.08:1 for everything, which it clearly isn't. I implemented a workaround so that the error goes away for the hexbin map.

We can't fully rely on automated tools for a11y anyways, so as long as we acknowledge of contrast issues like this, we should be fine.

**Comment by cfhull** [comment]
This ticket is "Completed" when it comes to implementing automated accessibility tooling. I will leave this it open though as a reminder to run any manual tools and general accessibility testing before we officially release.
